.mailmap

```diff
@@ -1544,3 +1544,4 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Øyvind Jensen <jensen.oyvind@gmail.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
+DayneSorvisto <daynesorvisto@yahoo.ca>
```

Comment:
you need to run the `.mailmap` file by the following command `python bin/mailmap_check.py` or read [here](https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file)